demos/python_demos/3d_segmentation_demo/3d_segmentation_demo.py

```diff
@@ -28,7 +28,7 @@
 from scipy.ndimage import interpolation
 from sys import stdout
 
-from openvino.inference_engine import IENetwork, IECore
+from openvino.inference_engine import IECore
```

Comment:
How do these changes in demo relate to quantizer tool?

Reply:
I think I opened the PR against `develop` by accident. It's actually supposed to go into `release`. Fixed now.